.github/workflows/test-matrix-2-levels.yml

```diff
@@ -0,0 +1,57 @@
+name: Test atmos-affected-stacks with 2 levels of nested matrices
```

Comment:
If I'm understanding this correctly, I believe this is trying to test with between 257-512 stacks, right? So there are 2 "chunks"? Consider updating the `nested matrices` term to `chunks` or `pages`?

Reply:
no. it means 256^2

Reply:
The workflow test that the GHA outputs correct json structure for 2 levels nested matrices support.

Reply:
Check this post to get about the pattern https://github.com/orgs/community/discussions/38704